src/app/opportunities-early-adopters/opportunities-early-adopters-page.component.html

```diff
@@ -62,7 +62,7 @@ <h1>Call for Applications for Early Adopters to Combat Online Gender-Based Haras
         effectiveness to their followers and networks.
       </p>
 
-      <h4>Who can apply?</h4>
+      <h4 class="title">Who can apply?</h4>
```

Comment:
I mean `class` would work but why not just style `h4` directly?

Reply:
MMh, I should have. Much simpler.